__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -539,7 +539,8 @@ describe('TESTING ChannelListSchema', () => {
             id: 'lastUpdatedBy'
           }
         },
-        targetName: 'name'
+        targetName: 'name',
+        targetImage: ''
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `targetImage` property is a good move, but it's currently set to an empty string. If this is the expected behavior, then it's fine. However, if there should be a default image or a null value, consider adjusting this accordingly.
```diff
-        targetImage: ''
+        targetImage: 'default_image_url' // or null
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->